specification/azuredata/resource-manager/readme.md

```diff
@@ -54,6 +54,7 @@ swagger-to-sdk:
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-ruby
+  - repo: azure-sdk-for-net
     after_scripts:
```

Comment:
Think this is now applying to the net sdk instead of the Ruby one

Reply:
@rajmusuku , please move one line up for `- repo: azure-sdk-for-net`